README.md

```diff
@@ -64,9 +64,33 @@ For building, `cmake` and its typical workflow is used.
 
     This will install all provided file to the specified location.
 
+### Setting up Julia
+After the library has been installed, you need to configure Julia for use with libtrixi. For
+this, create a directory where all necessary files will be placed, e.g., `libtrixi-julia`.
+Then, you can use the [`utils/libtrixi-init-julia`](utils/libtrixi-init-julia) tool to do
+the rest for you:
+```shell
+# Assuming you are in still in the `build/` directory inside the repo clone
+cd ..
+mkdir libtrixi-julia
+cd libtrixi-julia
+../utils/libtrixi-init-julia ..
+```
+When running a program that uses libtrixi, make sure the set up the `JULIA_DEPOT_PATH`
+environment variable to point to the `JULIA_DEPOT_LIBTRIXI` subfolder in the
+`libtrixi-julia` directory. In your code, pass the path to the `libtrixi-julia` directory as
```

Comment:
Yes. This part about the local depot  is not strictly necessary from a functional point of view. Looking forward, however, it will become more relevant from a performance point of view, where you want to be able to control the location of the depot to, eg, put it on a particular parallel file system.

The depot part is just for the location/storage of packages though. We still need to project dir (as documented here as well) to tell Julia *which* packages to actually use.